### PR TITLE
Resolve #1357, #1334, #1325, #1324

### LINKED ISSUE #1357: clear all storage data in resetBrowser

### PR DESCRIPTION
**Pull request by timothywangdev**

### CODE REVIEW

File: src/containers/RecipientsContainer.jsx [hunk 1]
```diff
@@ -43,10 +44,24 @@ class RecipientsContainer extends Component {
...
+    if (
+      (this.props.actionsPending.getRecipients || this.props.recipients) &&
+      !this.state.recipientsFetchStarted
+    ) {
+      // case 1. fetch recipients after mount
+      // use actionsPending to check fetch status
+      //
+      // case 2. accounts already fetched on other page
+      // use recipients prop to determine fetch status
+      this.setState({ recipientsFetchStarted: true })
```
Comment (InVincible2016) [inline src/containers/RecipientsContainer.jsx]: Why is this necessary? shouldn't be enough to just rely on this.props.actionsPending.getRecipients?

File: src/components/RecipientsComponent.jsx [hunk 2]
```diff
@@ -82,35 +84,55 @@ class RecipientsComponent extends Component {
...
+                    {!skeletonOnly ? (
+                      <UserAvatar
+                        style={{ width: 32 }}
+                        name={recipient.name}
+                        src={recipient.imageUrl}
+                      />
+                    ) : (
+                      <Skeleton
+                        variant='circle'
+                        width={32}
```
Comment (InVincible2016) [inline src/components/RecipientsComponent.jsx]: might well just put width and height to class too

File: src/tests/e2e/landing.test.e2e.js [hunk 3]
```diff
@@ -6,7 +6,8 @@ import { sleep } from './testUtils'
 const timeout = 180000
 
 describe('Landing page tests', () => {
-  beforeAll(async () => {
+  beforeEach(async () => {
+    await jestPuppeteer.resetBrowser()
```
Comment (InVincible2016) [inline src/tests/e2e/landing.test.e2e.js]: we shouldn't need to reset browser among each test in landing.

File: src/components/RecipientsComponent.jsx (hunk 2 above)
Comment (timothywangdev) [reply src/components/RecipientsComponent.jsx]: this is circle, different from other skeleton

File: src/containers/RecipientsContainer.jsx (hunk 1 above)
Comment (timothywangdev) [reply src/containers/RecipientsContainer.jsx]: gap between mount and actionsPending causing empty state being displayed as well as the add btn being enabled.

Comment (InVincible2016) [reply src/containers/RecipientsContainer.jsx]: We should consider moving getRecipient to postLoginPreparation.

File: src/tests/e2e/landing.test.e2e.js (hunk 3 above)
Comment (timothywangdev) [reply src/tests/e2e/landing.test.e2e.js]: yes, this will be another issue

File: src/containers/RecipientsContainer.jsx (hunk 1 above)
Comment (InVincible2016) [reply src/containers/RecipientsContainer.jsx]: The gap happens when the component is mounted but getRecipient is not dispatched. I think this can happen when the app is overloading by getAccounts and syncWIthNetwork.

Comment (timothywangdev) [reply src/containers/RecipientsContainer.jsx]: will be another issue
